docs/sources/datasources/_index.md

```diff
@@ -8,12 +8,17 @@ weight: 60
 
 # Data sources
 
-Grafana can query and integrate with many different types of databases. This is done by adding a **data source** of the type you want to query or integrate with.
-When you have created and configured a data source you are ready to start exploring and visualizing data, either in Explore or in a new Dashboard. A dashboard is composed of [panels]({{< relref "../panels-visualizations/" >}}), each panel contains a set of queries to one or more data sources.
+Grafana supports a wide range of _data sources_.
+These data sources might be files, services, or databases.
+You can also extend the supported data sources through one of the many [Plugins](/grafana/plugins/) (or [build your own](/docs/grafana/latest/developers/plugins/)).
```

Comment:
So this is a little nuanced, but you can't leverage a data source without a plugin. This makes it seem that plugins do something "extra" to data sources. 

Perhaps:
You can enable data sources by installing and configuring their associated Grafana plugins. For a list of available plugins, see [](). Alternatively, you can create your own, custom plugin. To learn more refer to []()

(I have a personal vendetta against unintroduced hyperlinks—feel free to ignore that)

Reply:
Hmm maybe I misunderstand. I thought Grafana had two models to use data sources:

- Built-in support. These ship with the Grafana binary, and they do not need to be installed.
- Extended through plugins. These have to be added from the available plugin list, or custom built.

Am I off here? I see that in some instances, the Grafana site says "plugin" to mean built-in data source.

> Loki Data Source - Native Plugin
> This is a built in data source that allows you to connect to the Loki logging service. [^1]

In my opinion, Grafana should avoid calling things that ship with the base binary a "plugin". I worry that this could corrode associations of meaning in the reader's heads, since, as far as I'm aware, plugin typically means an optional component to add to the system. It implies some customizability and tells people who don't want to use this data source that they can keep their binary footprint lighter by not installing it.

To call Loki a plugin implies that it could be plugged out, but that's not really true―it'd be more like rewiring.

I think saying things like "supported data source" or "built-in support" reduces cognitive reader load. To use _plugin_ to mean "outside of the base binary" creates a clear distinction. This is all to say that, if I haven't missed something, I'd prefer to try to reduce terms like "native plugin" in other places rather than introduce them here.

Does that make sense?

[^1]: https://grafana.com/grafana/plugins/loki/

Reply:
> This makes it seem that plugins do something "extra" to data sources.

I see your point about this sentence, though. What about this?

```diff
- You can also extend the supported data sources through one of the many [Plugins]
+ You can also add more data sources with Plugins
```

@Eve832 what do you think?  (ignoring the hyperlink―that's not a big deal to me).

Reply:
There are only like 13 data sources that we natively support (fewer in Cloud I think). There are 80+ data sources. All of those not supported require a plugin

Reply:
I just pushed that text to make the review more streamlined:
https://github.com/grafana/grafana/pull/61639/commits/4bd9fb7535d031e67b546716cc3e9e2d5f36a3d3

Reply:
Yeah, I this is def more clear